test/libsolidity/AnalysisFramework.h

```diff
@@ -33,7 +33,6 @@ namespace solidity::frontend
 {
 class Type;
 class FunctionType;
-using TypePointer = Type const*;
 using FunctionTypePointer = FunctionType const*;
```

Comment:
What about this?

Reply:
Well, this is in the tests, so not too important.

Reply:
It's not the `TypePointer` and the request was explicitly for that. I can replace `FunctionTypePointer` too if that is desired.
@chriseth also wanted to keep `TypePointers`.